opba-consent-rest-impl/src/main/java/de/adorsys/opba/consentapi/controller/ConsentServiceController.java

```diff
@@ -39,7 +41,7 @@ public CompletableFuture embeddedUsingPOST(
                         )
                         .scaAuthenticationData(body.getScaAuthenticationData())
                         .build()
-        ).thenApply(mapper::translate);
+        ).thenApply((FacadeResult<UpdateAuthBody> result) -> mapper.translate(result, null));
```

Comment:
Avoid using null and instead, add a translate function that calls no-op mapper. 
I.e. 
```
public <T> ResponseEntity<?> translate(FacadeResult<T> result) {
    return  translate(result, new NoOpMapper());
}
```
```
public <T> ResponseEntity<?> translate(FacadeResult<T> result, Mapper<T, ?> successMapper) {
        if (result instanceof FacadeRedirectErrorResult) {
            return handleError((FacadeRedirectErrorResult) result);
        }

        if (result instanceof FacadeResultRedirectable) {
            return handleRedirect((FacadeResultRedirectable) result);
        }

        if (result instanceof FacadeSuccessResult) {
            return successMapper.apply(handleSuccess((FacadeSuccessResult<T>) result));
        }

        throw new IllegalArgumentException("Unknown result type: " + result.getClass());
    }
```

Reply:
done